### PR TITLE
api support for attachment deletions

### PR DESCRIPTION
**Pull request by mvoloder**
API support for attachments deletion

method deletes uploaded files from local storage, but not from aws s3 storage

[Task link]http://the-shop.io:3000/projects/586016083e5bbe768349a4b0/sprints/5881d3aa3e5bbe274c5f1553/tasks/587a1e2f3e5bbe4334746f89

### CODE REVIEW

**Review by lotar — COMMENTED** [review]
This was supposed to be delete single attachment by ID route. Also did you test this since I don't see route definition in routes.

Just make sure this works as expected, I'll file new task for single attachment deletion

**Review by lotar — CHANGES_REQUESTED** [review]
Route definition missing

File: app/Http/routes.php [hunk 1]
```diff
@@ -53,6 +53,8 @@
         Route::get('configuration', 'ConfigurationController@getConfiguration');
         Route::post('email', 'EmailController@sendEmail');
         Route::post('upload', 'FileUploadController@uploadFile');
+        Route::get('upload/{id}', 'FileUploadController@getProjectUploads');
```
Comment (lotar) [inline app/Http/routes.php]: Didn't we say to have *just* the route for project uploads deletion?

File: app/Http/routes.php [hunk 2]
```diff
@@ -53,6 +53,8 @@
...
+        Route::delete('upload/{id}', 'FileUploadController@deleteProjectUploads');
```
Comment (lotar) [inline app/Http/routes.php]: make the route path to be `uploads/{projectId}/project`

**Review by lotar — CHANGES_REQUESTED** [review]

**Review by lotar — APPROVED** [review]

**Comment by lotar** [comment]
Why is this closed / branch deleted - it's not merged

**Comment by mvoloder** [comment]
restored deleted branch, resolved conflicts

**Pull request merged**